src/controller/CHIPDeviceController.cpp

```diff
@@ -1182,11 +1182,11 @@ void DeviceCommissioner::OnSessionEstablished()
 
     Device * device = &mActiveDevices[mDeviceBeingPaired];
 
-    mPairingSession.PeerConnection().SetPeerNodeId(device->GetDeviceId());
+    mPairingSession.SetPeerNodeId(device->GetDeviceId());
```

Comment:
Again, I would have expected the session to know its peer id way before this....

Reply:
Added a TODO tag.